File: code/src/test/scala/cern/colt/matrix/tlong/impl/SparseRCMLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseRCMLongMatrix2DViewTest(arg0: String) extends SparseRCMLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseRCMLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseRCMLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseRCMLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseRCMLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class SparseRCMLongMatrix2DViewTest extends SparseRCMLongMatrix2DTest {

    public SparseRCMLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCMLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCMLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCMLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
